packages/mui-material/src/Popover/Popover.js

```diff
@@ -233,6 +238,18 @@ const Popover = React.forwardRef(function Popover(inProps, ref) {
       const heightThreshold = containerWindow.innerHeight - marginThreshold;
       const widthThreshold = containerWindow.innerWidth - marginThreshold;
 
+      // if direction was RTL
+      let rightInRtlDirection =
+        containerWindow.innerWidth - elemTransformOrigin.horizontal - (left + anchorOffset.width);
+
+      if (anchorOrigin.horizontal === 'right') {
+        rightInRtlDirection += anchorOffset.width;
+      } else if (anchorOrigin.horizontal === 'center') {
+        rightInRtlDirection += anchorOffset.width;
+      } else {
+        rightInRtlDirection += anchorOffset.width;
+      }
```

Comment:
All three branches have exactly the same code.

Reply:
Oops! yes, you are right... I will remove this section of code and add anchorOffset.width to rightInRtlDirection without any condition